WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/WalletBalanceTileViewModel.cs

```diff
@@ -1,12 +1,21 @@
+using System.Reactive;
 using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Input;
+using Avalonia.VisualTree;
 using NBitcoin;
+using ReactiveUI;
 using WalletWasabi.Fluent.Extensions;
 using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.Tiles;
 
 public class WalletBalanceTileViewModel : ActivatableViewModel
 {
+	private readonly ObservableAsPropertyHelper<bool> _isCopyActive;
```

Comment:
@SuperJMN can we avoid using OAPH? I have personal beef with this character :)

Reply:
It seems that Avalonia doesn't like binding directly to ReactiveCommand.IsExecuting. The workaround I've applied works and no OAPH is required. Neat!